base/src/com/google/idea/blaze/base/wizard2/BlazeProjectImportBuilder.java

```diff
@@ -43,4 +57,31 @@ public List<Module> commit(
   public BlazeNewProjectBuilder builder() {
     return builder;
   }
+
+  @Override
+  public @Nullable Project createProject(String name, String path) {
+    Path currentPath = Path.of(path);
+    Path workspacePath = currentPath.getParent();
+    Path workspaceIdeaPath = workspacePath.resolve(Project.DIRECTORY_STORE_FOLDER);
+    File workspaceIdeaDir = workspaceIdeaPath.toFile();
+    // Allows checked in files to be used either using .idea under workspace root or checked in.
+    // createProject would delete the directory if it exists, so using loadProject instead.
+    if (workspaceIdeaDir.exists() && workspaceIdeaDir.isDirectory()) {
+      Path bazelProjectPath = currentPath.resolve(Project.DIRECTORY_STORE_FOLDER);
+      try {
+        FileUtil.copyDirContent(workspaceIdeaDir, bazelProjectPath.toFile());
```

Comment:
Can we make this conditional based on some registry value to not happen for everyone from now on?

Reply:
I was debating it...